src/constants/label.js

```diff
@@ -385,7 +442,10 @@ export const localizedNameWithLocalGloss = [
       // bother rendering it.
       ["concat", ["slice", ["var", "localizedName"], 0, 1], " "],
       { "font-scale": 0.001 },
-      listValuesExpression(["get", "name"], " \u2022 "),
+      listValuesExpression(["get", "name"], " \u2022 ", [
```

Comment:
Perhaps the bullet character could be extracted to a constant for better readability?